spacy/pipeline/entity_linker.py

```diff
@@ -345,7 +347,7 @@ def get_loss(self, examples: Iterable[Example], sentence_encodings: Floats2d):
         for eg in examples:
             kb_ids = eg.get_aligned("ENT_KB_ID", as_string=True)
 
-            for ent in eg.reference.ents:
+            for ent in eg.get_matching_ents():
```

Comment:
This is not related to this PR, but perhaps we could standardize on 
> for XXX in `examples`

and pick either `example`, `ex` or `eg`. I think most commonly across the repo, we'd use `eg`.

It just caught my attention because the first three edits in this PR use 3 different var names ;-)

Reply:
That's a good point. I'm not sure but I think:

- most existing code uses `eg`
- `example` is used in a specific context (maybe before me?)
- I tend to use `ex`

I would be fine with standardizing on `eg`, which is probably more common in existing code.

Reply:
I standardized on `eg` here.